device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-V256/buffers_defaults_t0.j2

```diff
@@ -0,0 +1 @@
+../ACS-SN5600/buffers_defaults_t0.j2
```

Comment:
we need to calculate this size as this is MSFT SKU and ACS-SN5600 is the generic one.